third_party/terraform/data_sources/data_source_dns_managed_zone.go

```diff
@@ -27,15 +27,19 @@ func dataSourceDnsManagedZone() *schema.Resource {
 			},
 
 			"name_servers": {
-				Type:     schema.TypeSet,
+				Type:     schema.TypeList,
```

Comment:
Why this change?

Reply:
`name_servers` is a `TypeList` in the dns_managed_zone resource.

Between the time when this test last passed and now we changed from using a custom comparison method within this class to `checkDataSourceStateMatchesResourceState`. 

Comparison with the `checkDataSourceStateMatchesResourceState` method fails (I believe because of the different type) even though the list/set contain the same items. I believe because the new comparison compares on every attribute of the resource: https://github.com/terraform-providers/terraform-provider-google/blob/d0d6a895805f97bdf2a53197d325a2cf19f68c9b/google/test_utils.go#L110

while the old one only compared the whole `name_servers` object: https://github.com/terraform-providers/terraform-provider-google/blob/c605b0bc4ba2291d86b1038ce4b45b5ea50c1bb9/google/data_source_dns_managed_zone_test.go#L43